src/api/liveSpecsExt.ts

```diff
@@ -429,6 +430,44 @@ const getLiveSpecShards = (tenant: string, entityType: Entity) => {
         .eq('spec_type', entityType);
 };
 
+export interface TrialCollectionQuery {
+    catalog_name: string;
+    updated_at: string;
+}
+
+const getTrialCollections = async (prefixes: string[]) => {
```

Comment:
Gonna "invert" this approach. Noting here in writing since we discussed on a video chat.